admin_guide/persistent_storage/persistent_storage_cinder.adoc

```diff
@@ -0,0 +1,85 @@
+= Persistent Storage Using OpenStack Cinder volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+You can provision your OpenShift cluster with
+link:../../architecture/additional_concepts/storage.html[persistent storage] using
+https://access.redhat.com/documentation/en/red-hat-enterprise-linux-openstack-platform/version-7/red-hat-enterprise-linux-openstack-platform-7-architecture-guide/chapter-1-components#comp-cinder[OpenStack Cinder].
+Some familiarity with Kubernetes and OpenStack is assumed.
+
+[NOTE]
+====
+OpenShift needs to be properly configured for OpenStack, see chapter
+link:../configuring_openstack.html[Configuring OpenShift on OpenStack]
+.
+====
+
+The Kubernetes link:../../dev_guide/persistent_volumes.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage
+and gives users a way to request those resources without having any knowledge of
+the underlying infrastructure.
+
+For a detailed example, see the guide for
+https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
+MySQL using persistent volumes].
```

Comment:
I have OpenShift PR, which adds Cinder (and Fibre Channel): https://github.com/openshift/origin/pull/4403